improver/lightning.py

```diff
@@ -204,3 +204,220 @@ def latitude_to_threshold(
         rescale(latitude, (50.0, 10), (midlatitude, tropics), clip=True),
         rescale(latitude, (-50.0, -10), (midlatitude, tropics), clip=True),
     )
+
+
+class LightningMultivariateProbability(PostProcessingPlugin):
+    """
+    The algorithm outputs the probability of at least one lightning strike within
+    20 km of the location in a three hour period. The probabilities are calculated
+    for each individual ensemble member, and then they are averaged for the ensemble
+    probability forecast.
+
+    Inputs:
+    CAPE (cape; calculated with water loading and latent heat of freezing estimated)
+    Lifted Index (liftidx)
+    Precipitable Water (pwat in mm)
+    CIN (cin)
+    3-hour Accumulated Precipitation (apcp in inches)
+    Outputs:
+    20 km lightning probability over the valid time of the accumulated precipitation
+
+    ---------------------------+------------+----------------------------------------
+
+    Regression equation when CAPE and APCP are greater than zero:
+    lprob= cape*APCP
+    lprob=0.13*alog(lprob+0.7)+0.05
+
+    If APCP is very low, a separate regression equation is used to predict lightning probability.
+    The definition of “very low” is raised slightly when PWAT values are high. This is because
+    the model often produces showery precipitation that doesn’t access the actual instability in
+    very moist environments:
+    lprob_noprecip=CAPE/(CIN+100.0)
+    lprob_noprecip=0.025*alog(lprob_noprecip+0.31)+0.03
```

Comment:
Changed as suggested in the two locations you point out. Good catch!

```
    Regression equation when CAPE and APCP are greater than zero:
    lprob= cape*APCP
    lprob=0.13*ln(lprob+0.7)+0.05

    If APCP is very low, a separate regression equation is used to predict lightning probability.
    The definition of “very low” is raised slightly when PWAT values are high. This is because
    the model often produces showery precipitation that doesn’t access the actual instability in
    very moist environments:
    lprob_noprecip=CAPE/(CIN+100.0)
    lprob_noprecip=0.025*ln(lprob_noprecip+0.31)+0.03
    APCP=APCP-(PWAT/1000)
```